docs/root/configuration/cluster_manager/cluster_stats.rst

```diff
@@ -51,6 +51,7 @@ Every cluster has a statistics tree rooted at *cluster.<name>.* with the followi
   upstream_cx_destroy_with_active_rq, Counter, Total connections destroyed with 1+ active request
   upstream_cx_destroy_local_with_active_rq, Counter, Total connections destroyed locally with 1+ active request
   upstream_cx_destroy_remote_with_active_rq, Counter, Total connections destroyed remotely with 1+ active request
+  upstream_cx_destroy_after_draining_rq, Counter, Total connections destroyed after being drained of active requests
```

Comment:
Unless we are adding this everywhere, can we potentially just add a redis cluster/filter specific stat for this? We are trying to cut back on generically adding stats to cluster stats.

Reply:
I'll look into a Redis cluster specific stats, but since there are none so far, I'll have to work out how to do that...